packages/connect/src/api/firmware/getBinary.ts

```diff
@@ -1,26 +1,22 @@
-import type { FirmwareRelease } from '@trezor/device-utils';
-
 import { httpRequest } from '../../utils/assets';
 
 const ALL_SLASHES_AT_THE_END_REGEX = /\/+$/;
```

Comment:
this regex can be found also here https://github.com/trezor/trezor-suite/blob/feat/firmware-update/packages/suite-desktop-core/src/libs/update-checker.ts#L18
but I am not sure if there is a meaningful way to share it

Reply:
What about something like this https://github.com/trezor/trezor-suite/pull/19109 ?